src/Pages/MembershipApplication/MembershipForm.js

```diff
@@ -52,6 +55,16 @@ export default function MembershipForm(props) {
       handleChange: e => setPassword(e.target.value)
     }
   ];
+  const [semesterImage, setSemesterImage] = useState({
```

Comment:
same thing with semester image here as the other comment i left. If you see yourself reusing the same code, consider moving the `getSemesterImage` function to another file in `util` or `APIFunctions`.

Reply:
I just changed it to the tech background that Annabel found